components/neeva/renderer/content_filtering_agent.cc

```diff
@@ -51,11 +51,19 @@ std::unique_ptr<base::MemoryMappedFile> MapRegion(
 ContentFilteringAgent::ContentFilteringAgent(
     blink::ThreadSafeBrowserInterfaceBrokerProxy* broker)
     : task_runner_(base::SequencedTaskRunnerHandle::Get()) {
+
+  // Binds the service as a remote and creates a pipe to the Browser.
+  // See ContentFilteringService::AddInterface() for the handling of the reciever part. 
+  // @Darin why is the Browser broker when registering the Renderer service pipe??
   broker->GetInterface(service_.BindNewPipeAndPassReceiver());
 
+  // Allows the ContentFilterConfig to call OnReceiveNewRules() and 
```

Comment:
I think this comment is a little self-evident. It also runs the risk of becoming stale if `ContentFilterConfig` gets renamed. Is this really helpful?

Reply:
I feel like it is to an outsider non-Chromium viewer. Now that I understand the code, it is totally self-evident, but I still find it useful if I were to revisit this in 6 months